packages/ai-constructs/src/conversation/conversation_handler_construct.test.ts

```diff
@@ -222,4 +222,67 @@ void describe('Conversation Handler Function construct', () => {
       Handler: 'index.handler',
     });
   });
+
+  void describe('memory property', () => {
+    void it('sets valid memory', () => {
+      const app = new App();
+      const stack = new Stack(app);
+      new ConversationHandlerFunction(stack, 'conversationHandler', {
+        models: [],
+        memoryMB: 234,
+      });
+      const template = Template.fromStack(stack);
+
+      template.hasResourceProperties('AWS::Lambda::Function', {
+        MemorySize: 234,
+      });
+    });
+
+    void it('sets default memory', () => {
+      const app = new App();
+      const stack = new Stack(app);
+      new ConversationHandlerFunction(stack, 'conversationHandler', {
+        models: [],
+        memoryMB: 512,
+      });
```

Comment:
```suggestion
      new ConversationHandlerFunction(stack, 'conversationHandler', {
        models: [],
      });
```
We should remove this to test that the default memory is set.

Reply:
good catch. updated.